tests/firebase_api/test_firebase_data_client.py

```diff
@@ -0,0 +1,378 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+import pytest
+from django.core.cache import cache
+from django.core.exceptions import ImproperlyConfigured
+
+from integreat_cms.firebase_api.firebase_data_client import FirebaseDataClient
+
+if TYPE_CHECKING:
+    from pytest_django.fixtures import SettingsWrapper
+    from requests_mock.mocker import Mocker
+
+
+class TestFirebaseDataClient:
+    """
+    Test for :class:`~integreat_cms.firebase_api.firebase_api_client.FirebaseDataClient`
+    """
+
+    def teardown_method(self) -> None:
+        cache.delete("firebase_data")
```

Comment:
This also needs to be removed, because this feature won't be used in the GUI at the moment and as this functionality will be called once per day there is no cache anymore anyway